nailgun/entity_mixins.py

```diff
@@ -191,6 +196,15 @@ def _payload(fields, values):
                 values[field_name + '_ids'] = [
                     entity.id for entity in values.pop(field_name)
                 ]
+            elif isinstance(field, ListField):
+                def parse(obj):
```

Comment:
Coveralls says this inner function is not covered, can you write a simple test case that covers this elif, or if does not make sense to test include a `... ListField): # pragma: no cover` comment here